### PR TITLE
Feature/timber logging

### PR DESCRIPTION
**Pull request by EarlOfEgo**
Since Proguard is disabled for the release build all log calls are shown when the app is executed. Do `adb shell logcat | grep xmrwallet` to see them. This is not good, no log information should be enabled for the release build, only for debug builds. (Or even worse as it is stated in the Timber Readme: "…every time you log in production, a puppy dies.") 
I added Timber that makes logging easy and later can be extended when we add crash reporting. And of course now nothing is logged in production.

### CODE REVIEW

File: app/src/main/java/com/m2049r/xmrwallet/ReceiveFragment.java [hunk 1]
```diff
@@ -352,7 +352,7 @@ public Bitmap generate(String text, int width, int height) {
                 }
             }
             Bitmap bitmap = Bitmap.createBitmap(pixels, 0, width, width, height, Bitmap.Config.RGB_565);
-            bitmap = addLogo(bitmap);
+            bitmap = addTimbero(bitmap);
```
Comment (m2049r) [inline app/src/main/java/com/m2049r/xmrwallet/ReceiveFragment.java]: oops!

File: app/src/main/java/com/m2049r/xmrwallet/ReceiveFragment.java [hunk 2]
```diff
@@ -361,8 +361,8 @@ public Bitmap generate(String text, int width, int height) {
     }
 
     // TODO check if we can sensibly cache some of this
-    private Bitmap addLogo(Bitmap qrBitmap) {
-        Bitmap logo = getMoneroLogo();
+    private Bitmap addTimbero(Bitmap qrBitmap) {
+        Bitmap logo = getMoneroTimbero();
```
Comment (m2049r) [inline app/src/main/java/com/m2049r/xmrwallet/ReceiveFragment.java]: at least it's consistent :)

File: app/src/main/java/com/m2049r/xmrwallet/ReceiveFragment.java [hunk 3]
```diff
@@ -386,7 +386,7 @@ private Bitmap addLogo(Bitmap qrBitmap) {
 
     private Bitmap logo = null;
 
-    private Bitmap getMoneroLogo() {
+    private Bitmap getMoneroTimbero() {
```
Comment (m2049r) [inline app/src/main/java/com/m2049r/xmrwallet/ReceiveFragment.java]: Timbero...

File: app/src/main/java/com/m2049r/xmrwallet/SendFragment.java [hunk 4]
```diff
@@ -352,7 +352,7 @@ private void prepareSend() {
         int mixin = Mixins[sMixin.getSelectedItemPosition()];
         int priorityIndex = sPriority.getSelectedItemPosition();
         PendingTransaction.Priority priority = Priorities[priorityIndex];
-        Log.d(TAG, dst_addr + "/" + paymentId + "/" + amount + "/" + mixin + "/" + priority.toString());
+        Timber.d(dst_addr + "/" + paymentId + "/" + amount + "/" + mixin + "/" + priority.toString());
```
Comment (m2049r) [inline app/src/main/java/com/m2049r/xmrwallet/SendFragment.java]: Why not
```Java
Timber.d("%s/%s/%s/%s/%s", dst_addr, paymentId, amount, mixin, priority.toString());
```
?
In general many instances use %s and many others (still) use +. We should try to be consistent, sooner or later,

**Review by m2049r — CHANGES_REQUESTED** [review]
Excellent idea. Some find/replace bugs.

File: app/src/main/java/com/m2049r/xmrwallet/ReceiveFragment.java (hunk 1 above)
Comment (EarlOfEgo) [reply app/src/main/java/com/m2049r/xmrwallet/ReceiveFragment.java]: haha, yeah I will change it

File: app/src/main/java/com/m2049r/xmrwallet/SendFragment.java (hunk 4 above)
Comment (EarlOfEgo) [reply app/src/main/java/com/m2049r/xmrwallet/SendFragment.java]: Reason is that Timber provides a Lint rule for `"bla " + value` but not for `value + " " + value + " "` but you are right it should be consistent and I will change it.

**Review by m2049r — APPROVED** [review]

**Pull request merged**